cli/lsp/language_server.rs

```diff
@@ -3121,7 +3121,7 @@ impl Inner {
       .performance
       .mark("virtual_text_document", Some(&params));
     let specifier = self.url_map.normalize_url(&params.text_document.uri);
-    let contents = if specifier.as_str() == "deno:/status.md" {
+    let contents = if specifier.as_str() == "internal:/status.md" {
```

Comment:
This breaks vscode_deno, which does a request to this.

Reply:
https://github.com/denoland/vscode_deno/blob/4b2787824f03879e2cbbff47c9921110ce1f20fb/client/src/commands.ts#L254